analytics/population/nc_updated_pop_comparison.ipynb

```diff
@@ -0,0 +1,991 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    NC_pop_geopandas['NAME'] = NC_pop_geopandas['NAME'].str.replace(' NC', '')
</pre><p>We're repeating a lot of code here. I think we should create a dictionary mapping old names to new names, and then use a helper function that calls str.replace for each old name/new name pair.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hackforla/311-data/pull/1336/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Thanks for the suggestion.  Defined a function to replace the old names with the new ones. </p>